contracts/libs/Actions.sol

```diff
@@ -84,4 +93,19 @@ library Actions {
                 amount: _args.amount
             });
     }
+
+    /**
+     * @notice Parses the passed in action argmuents to get the argmuents for a settle vault action
+     * @param _args The general action arguments structure
+     * @return The arguments for a settle vault action
+     */
+    function _parseSettleVaultArgs(ActionArgs memory _args) internal returns (SettleVaultArgs memory) {
```

Comment:
should this check if the sender is the vault owner or operator ? or do we do this check somewhere else ?

Reply:
I think somewhere else because the library won't have access to the operator mapping right?

Reply:
no that check is in the controller.